target-chains/cosmwasm/tools/src/pyth_config.ts

```diff
@@ -128,4 +128,41 @@ export const CONFIG: Config = {
       denom: "inj",
     },
   },
+  // this details below are invalid and random
```

Comment:
Can't we abstract common configurations such as data and governance sources to a common place for testnet/mainnet and use them without repeating the logic?

Reply:
Took the common configurations out.